packages/governance/src/electorateTools.js

```diff
@@ -69,7 +70,12 @@ const getQuestion = (questionHandleP, questionStore) =>
  */
 const getPoserInvitation = (zcf, addQuestion) => {
   const questionPoserHandler = () => Far(`questionPoser`, { addQuestion });
-  return zcf.makeInvitation(questionPoserHandler, `questionPoser`);
+  return zcf.makeInvitation(
+    questionPoserHandler,
+    `questionPoser`,
+    undefined,
+    EmptyProposal,
```

Comment:
This one is [for the use of the contractGoveror. ](https://github.com/Agoric/agoric-sdk/blob/ffe5e7ff3dd7ec1ffbbcd86e244ef167ed70c53b/packages/governance/src/contractGovernor.js#L195).  Always empty.